### PR TITLE
Add blog post covering infrastructure subcomittee 2018/02 meeting

### PR DESCRIPTION
**Pull request by rgaiacs**
@fmichonneau you might want to review this before it goes live.

A copy of this post should be publish at http://www.datacarpentry.org/.

### CODE REVIEW

**Comment by weaverbel** [comment]
Thanks @rgaiacs I will put a copy on DC once @fmichonneau has given the green light

File: _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+category: ["Community", "Lesson Infrastructure Subcommittee"]
+---
+
+On 14 February 2018 at 18:00UTC+0,
+the Lesson Infrastructure Subcommittee had their 2018 February meeting.
+This post will cover the topics discussed and their resolutions.
+
+## Software Carpentry and Data Carpentry merge
+
+Software Carpentry and Data Carpentry had merge.
```
Comment (fmichonneau) [inline _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md]: have merged.

File: _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+Their [Executive Council was elected](https://software-carpentry.org/blog/2017/12/executive-council-2018.html),
+they have a [new fiscal sponsor](https://software-carpentry.org/blog/2018/01/fiscal-sponsor-transition.html)
+and
+a [new logo](https://software-carpentry.org/blog/2018/02/our-new-logo.html).
+Also,
+they have [hired Dr. François Michonneau](http://www.datacarpentry.org/blog/curriculum-dev-scaling/) to lead curriculum development efforts.
+
+This all are very exciting news. And we even didn't finished February.
```
Comment (fmichonneau) [inline _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md]: And February isn't over yet.

File: _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+## Windows Installer
+
+We will proceed to adopt Git for Windows v2.15.1,
+which includes nano,
+and drop our Windows installer.
+This should simplify our installation instructions
+and
+provide a better experience for our learners during the workshop.
+All the details of this changes are covered in [this document](https://github.com/carpentries/lesson-infrastructure/blob/master/liep/001-git-for-windows-nano.md)
+that summarise many pull requests and issues over the last 6 months.
```
Comment (fmichonneau) [inline _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md]: which summarises

File: _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+## Workshop Installation Instructions
+
+In January,
+a [blog post was published](https://software-carpentry.org/blog/2018/01/proposal-workshop-template.html)
+with a proposal to enhancement the process to customize the installation instructions at the workshop page.
```
Comment (fmichonneau) [inline _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md]: to enhance the process of customizing the installation instructions on the workshop page.

File: _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+After a initial round of feedback,
```
Comment (fmichonneau) [inline _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md]: After an initial

File: _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+Raniere Silva,
+François Michonneau
+and
+Rémi Emonet
+will move the proof of concept included in the proposal forward to a feasibility prototype.
+The plan is to have the prototype ready in April
+and,
+if we decide to go forward,
+made it available to instructors in July
```
Comment (fmichonneau) [inline _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md]: to make it

File: _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+## Labels and Lessons
+
+If you don't know,
+[GitHub rolled out some improvements to labels](https://github.com/blog/2505-label-improvements-emoji-descriptions-and-more).
+This changes couldn't be in a better time.
+Erin Becker
+and
+François Michonneau
+are working to revamp the labels used in the Git repositories that host the lessons.
+The new set of labels will facilitate for instructors and learners to navigate on issues and pull requests.
```
Comment (fmichonneau) [inline _posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md]: will facilitate the navigation around issues and pull requests for maintainers and contributors.

**Review by fmichonneau — COMMENTED** [review]
Thanks for putting this together Raniere! I included some suggestions for edits. Otherwise, the content reflects accurately the meeting.

**Comment by weaverbel** [comment]
Hey @rgaiacs - can you please review @fmichonneau 's suggestions and then I can merge?

**Comment by rgaiacs** [comment]
@fmichonneau Thanks for the suggestions.

> then I can merge?

@weaverbel Should be good to merge. Thanks.

**Comment by jduckles** [comment]
Merging.

**Pull request merged**